mods/bones/init.lua

```diff
@@ -153,80 +153,79 @@ local function may_replace(pos, player)
 	return node_definition.buildable_to and not minetest.is_protected(pos, player:get_player_name())
 end
 
-minetest.register_on_dieplayer(function(player)
-	if minetest.setting_getbool("creative_mode") then
-		return
-	end
-	
-	local player_inv = player:get_inventory()
-	if player_inv:is_empty("main") and
-		player_inv:is_empty("craft") then
-		return
-	end
+if not minetest.setting_getbool("disable_bones") and not minetest.setting_getbool("creative_mode") then
```

Comment:
> Instead of indenting the whole file just do this:

No, that results in registering a function for no reason, that 'returns' as it's first action, just for the sake of saving one indentation.